src/index.html

```diff
@@ -11,7 +11,121 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post">
+      <fieldset class="block-questions">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="form-field">Surname:
+          <input type="text"
+          name="Personal information"
+          id="surname"
+          autocomplete="off">
```

Comment:
```suggestion
          <input
            type="text"
            name="Personal information"
            id="surname"
            autocomplete="off"
          >
```
Форматування неправильне. Виправ скрізь.
Почитати можна тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation

Reply:
Готово